src/Display/Accordion/Accordion.test.tsx

```diff
@@ -0,0 +1,89 @@
+import '@testing-library/jest-dom/extend-expect';
+import * as React from 'react';
+import { render } from '@testing-library/react';
+
+import Accordion, { IconPosition } from './Accordion';
+import userEvent from '@testing-library/user-event';
+
+describe('<Accordion /> prop className', () => {
+  const matchSnapshot = (className: string) => {
+    test(`should match snapshot when class name ${className} is passed`, () => {
+      const { asFragment } = render(
+        <Accordion className={className}>
+          <Accordion.Panel
+            className={className}
+            label="label"
+            content="content"
+          />
+        </Accordion>
+      );
+      // validate class name from snapshot
+      expect(asFragment()).toMatchSnapshot();
+    });
+  };
+
+  ['test', undefined].forEach(className => matchSnapshot(className));
```

Comment:
```suggestion
  ['test', undefined].forEach(matchSnapshot);
```

Minor tip: In these cases you can just pass the function instead of creating a new function that does exactly the same thing as the wrapped function.

Reply:
Updated!